common.sh

```diff
@@ -14,23 +17,26 @@ else
   HOMEBREW_PREFIX=/opt/homebrew
 fi
 
-cbr() {
+f5m_configure() {
   # Install plugins to /usr/local for both arm and x86 to ease fcitx5 search
-  PKG_CONFIG_PATH=$HOMEBREW_PREFIX/lib/pkgconfig cmake -B build -G Ninja \
+  PKG_CONFIG_PATH="$HOMEBREW_PREFIX/lib/pkgconfig:$PKG_CONFIG_PATH" cmake -B build -G Ninja \
     -DCMAKE_BUILD_TYPE=Release \
     -DCMAKE_FIND_ROOT_PATH="/Library/Input Methods/Fcitx5.app/Contents;$HOMEBREW_PREFIX" \
-    -DCMAKE_OSX_ARCHITECTURES=$ARCH "$@"
+    -DCMAKE_INSTALL_RPATH="/Library/Input Methods/Fcitx5-x86_64.app/Contents/lib" \
+    -DCMAKE_OSX_ARCHITECTURES=$ARCH \
+    -DCMAKE_INSTALL_PREFIX="$ADDON_INSTALL_PREFIX" \
+    "$@"
 }
 
-cb() {
+f5m_build() {
   cmake --build build
 }
 
-ci() {
+f5m_install() {
   cmake --install build
 }
 
-tbz() {
-  cd $DESTDIR/..
-  tar cjvf $name-$ARCH.tar.bz2 -C $name/usr/local lib share
+f5m_make_tarball() {
+  cd $ADDON_INSTALL_PREFIX/..
+  tar cjvf $name-$ARCH.tar.bz2 -C $name .
```

Comment:
This needs revert. Using `.` instead of subdir names requires write access to /usr when extracting, unless `--strip-components=1`. Since extracting is a user action, I don't want to complicate it with one more cli parameter.
It can be more intelligent to only include existing directories.

Reply:
done